eclass/meson.eclass

```diff
@@ -35,31 +35,24 @@
 # @CODE
 
 case ${EAPI:-0} in
```

Comment:
```suggestion
case ${EAPI} in
```

Reply:
Updated.  (This is another case where it seems most other eclasses are doing that.)